sdk/tables/Azure.Data.Tables/src/TableClient.cs

```diff
@@ -1155,20 +1236,179 @@ public virtual Response SetAccessPolicy(IEnumerable<SignedIdentifier> tableAcl,
         public static string CreateQueryFilter(FormattableString filter) => TableOdataFilter.Create(filter);
 
         /// <summary>
-        /// Create a <see cref="TableTransactionalBatch" /> for the given <paramref name="partitionKey"/> value.
+        /// Submits the batch transaction to the service for execution.
+        /// The sub-operations contained in the batch will either succeed or fail together as a transaction.
         /// </summary>
-        /// <param name="partitionKey">The partitionKey context for the batch.</param>
-        /// <returns>An instance of <see cref="TableTransactionalBatch"/>.</returns>
-        public virtual TableTransactionalBatch CreateTransactionalBatch(string partitionKey)
+        /// <param name="batchOperations">The <see cref="IEnumerable{T}"/> containing the <see cref="BatchItem"/>s to submit to the service.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        /// <returns><see cref="Response{T}"/> containing a <see cref="TableBatchResponse"/>.</returns>
+        /// <exception cref="RequestFailedException"/> if the batch transaction fails./>
+        /// <exception cref="InvalidOperationException"/> if the batch has been previously submitted.
+        public virtual async Task<Response<TableBatchResponse>> SubmitTransactionAsync(
```

Comment:
Should we consider using the word "batch" in the name?  I wouldn't think to connect a batch with "transaction" at a glance if I were scanning docs/intellisense.   Maybe something like `SubmitBatchTransactionAsync` ?